src/util/index.js

```diff
@@ -16,3 +17,25 @@ export const ogMetaTags = (title, description, image) => {
 }
 
 export const isInCommunity = ({ pathname }) => pathname.startsWith('/c/')
+
+export const findChildLink = element => {
+  if (element.nodeName === 'A') return element
+  if (element.hasChildNodes()) {
+    const mappedNodes = []
+    for (var i = 0; i < element.childNodes.length; i++) {
+      mappedNodes.push(findChildLink(element.childNodes[i]))
+    }
+    return filter(id => id, mappedNodes)[0]
+  }
+  return false
+}
+
+export const fireEvent = (el, etype) => {
```

Comment:
let's call this `dispatchEvent` instead since that's the name of the more standard method